documentation/user_docs/docs/manual/Advanced_Multifit.rst

```diff
@@ -2,33 +2,73 @@
 Advanced Multifit
 #################
 
-This document describes how to set up an object to work with Multifit specifications
+Parallel fitting
+================
+
+It is possible to use ``multifit`` and its derivatives (``tobyfit``, ``multifit_sqw``, ``multifit_sqw_sqw``) in parallel
+by either enabling HPC options through
+
+::
+
+   >> hpc('on');
+
+or by setting ``parallel_multifit`` directly in the ``hpc_config``
+
+::
+
+   >> hc = hpc_config;
+   >> hc.parallel_multifit = true;
+
+Parallel multifit decomposes the objects passed in into slices which are distributed between the processors. I.e. if we
+are fitting three ``IX_dataset`` objects with 100 points between two processors, each processor will receive 50
+points from each ``IX_dataset``.
+
+This decomposition is performed handled differently for each of the three classes of fittable objects:
+- For sqw objects, the splitting occurs across pixels. (``N/P`` pixels per process)
```

Comment:
It's not clear to me here and below what the N/P and B/P are for - they don't appear to be referred to below. Maybe better say: "In each case the data are divided into NP items per process, for sqw items are pixels, for dnd items are bins and for ix_dataset items are data points